src/components/LoggedInFeature/LoginRequiredModal.messages.js

```diff
@@ -0,0 +1,17 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  featureBlockedTitle: {
+    id: 'cboard.components.LoginRequiredModal.featureBlockedTitle',
+    defaultMessage: 'This feature is blocked'
```

Comment:
please replace this text with something similar than:
'Log in to use this feature'